views/info/filters.pug

```diff
@@ -8,76 +8,96 @@ block content
-          p #[code o:"destroy target creature"] cards whose oracle text contains #[code destroy target creature].
-          p You can also use parentheses.
-          p #[code t:instant OR (t:creature o:flash)] cards which are instants, or cards which are creatures with flash.
-          p #[code (t:artifact t:creature) OR (t:instant OR t:sorcery)] cards which are instants or sorceries, or cards which are artifact creatures.
-          p You can put - before anything to negate it.
+        #collapseGeneral.collapse.show(aria-labelledby="general-syntax" data-parent="#syntax-accordion")
+          p You can combine any number of filters together using #[code AND] or #[code OR].
+          p #[strong Examples:]
+          table.table
+            tr
+              td(scope="col") #[code t:instant OR t:sorcery]
+              td(scope="col") Cards that are either instants or sorceries.
+            tr
+              td(scope="col") #[code t:instant t:tribal]
+              td(scope="col") Cards that are both instants and tribal.
+          p You can use quotes to ignore spaces.
+          p #[strong Examples:]
+          table.table
+            tr
+              td(scope="col") #[code goblin blood]
+              td(scope="col") Cards whose names contain both "blood" and "goblin".
+            tr
+              td(scope="col") #[code "goblin blood"]
+              td(scope="col") Cards whose names contain exactly "goblin blood".
+            tr
+              td(scope="col") #[code o:destroy o:target o:creature] 
+              td(scope="col") Cards whose oracle text contains each of "destroy", "target", and "creature".
+            tr
+              td(scope="col") #[code o:"destroy target creature"]
+              td(scope="col") Cards whose oracle text contains exactly "destroy target creature".
+          p You can also use parentheses to combine clauses.
+          p #[strong Examples:]
+          table.table
+            tr
+              td(scope="col") #[code t:instant OR (t:creature o:flash)]
+              td(scope="col") Cards which are instants, or cards which are creatures with flash.
+            tr
+              td(scope="col") #[code (t:artifact t:creature) OR (-t:creature o:create)]
+              td(scope="col") Cards which are artifact creatures, or cards that aren't creatures and have "create" in their oracle text.
+          p You can put #[code -] before anything to negate it.
           p #[strong Examples:]
           table.table
             tr
               td(scope="col") #[code -c:w] 
-              td(scope="col") Cards that are not white
+              td(scope="col") Cards that are not white.
             tr
               td(scope="col") #[code -o:draw] 
-              td(scope="col") Cards which do not have draw in their oracle text
+              td(scope="col") Cards which do not have draw in their oracle text.
             tr
               td(scope="col") #[code -bob] 
```

Comment:
This may not work currently.

Reply:
Tested, it doesn't work.  I changed it to a different example.  Is this intended behavior?  Should I file a bug?

Reply:
#1352 created